Hcal/include/Hcal/HcalDoubleEndRecProducer.h

```diff
@@ -31,13 +40,7 @@ class HcalDoubleEndRecProducer : public framework::Producer {
   /// length of clock cycle [ns]
   double clock_cycle_;
 
- public:
-  HcalDoubleEndRecProducer(const std::string& n, framework::Process& p)
-      : Producer(n, p) {}
-
-  virtual ~HcalDoubleEndRecProducer() = default;
-  void configure(framework::config::Parameters& p) override;
-  void produce(framework::Event& event) override;
+  enableLogging("HcalDoubleEndRecProducer")
```

Comment:
You do not need to `enableLogging` in event processors. The base class already creates `theLog_`.